### PR TITLE
warning incorrect threshold config

### PR DESCRIPTION
**Pull request by dmthuc**
Give a warning when config for load threshold is invalid.
```
{"message": "load_threshold in prod env must be less than 1, current value: 2", "level": "WARNING", "name": "livekit.agents", "timestamp": "2024-10-12T08:20:02.168613+00:00"}
```
This pr is for issue https://github.com/livekit/agents/issues/884

### CODE REVIEW

File: livekit-agents/livekit/agents/cli/cli.py [hunk 1]
```diff
@@ -234,6 +234,7 @@ async def _run_loop():
 
 def run_worker(args: proto.CliArgs) -> None:
     setup_logging(args.log_level, args.devmode)
+    args.validate_config()
```
Comment (davidzhao) [inline livekit-agents/livekit/agents/cli/cli.py:237]: it'd be better to make the call directly vs another abstraction

```suggestion
    args.opts.validate_config()
```

File: livekit-agents/livekit/agents/cli/proto.py [hunk 2]
```diff
@@ -30,6 +30,9 @@ class CliArgs:
     # when reload/dev mode is enabled
     mp_cch: socket.socket | None = None
 
+    def validate_config(self):
+        self.opts.validate_config(self.devmode)
+
```
Comment (davidzhao) [inline livekit-agents/livekit/agents/cli/proto.py:35]: this wouldn't be needed

File: livekit-agents/livekit/agents/worker.py [hunk 3]
```diff
@@ -185,6 +185,14 @@ class WorkerOptions:
     The HTTP server is used as a health check endpoint.
     """
 
+    def validate_config(self, devmode: bool):
+        if not devmode:
+            load_threshold = _WorkerEnvOption.getvalue(self.load_threshold, False)
```
Comment (davidzhao) [inline livekit-agents/livekit/agents/worker.py:190]:
```suggestion
        load_threshold = _WorkerEnvOption.getvalue(self.load_threshold, devmode)
```

File: livekit-agents/livekit/agents/cli/cli.py (hunk 1 above)
Comment (dmthuc) [reply livekit-agents/livekit/agents/cli/cli.py]: I've updated according to your advices

**Review by davidzhao — APPROVED** [review]

**Pull request merged**